### DISCUSSION TITLE
从节点不工作，监控面板上没有消费消息

### DISCUSSION THREAD

**Discussion by xcfuu** [apache/rocketmq#6218] [Q&A]
![image](https://user-images.githubusercontent.com/29519964/222093620-cf94fea9-f80d-490f-b4de-7b6375b2f35c.png)
![image](https://user-images.githubusercontent.com/29519964/222093795-899a08c4-1e07-4f42-81db-61acdefa8355.png)
版本
rocketmq-broker-4.2.0-incubating-SNAPSHOT.jar

请问这种情况应该从什么方向去查问题？

**Comment by RongtongJin** [comment]
@xcfuu HA相关日志在~/logs/rocketmqlogs/store.log，可以看下

**Reply by xcfuu** [reply]
![image](https://user-images.githubusercontent.com/29519964/222305501-8492bd3c-5c02-4777-867d-eb97ef9e8fe6.png)
基本都是这些日志

**Reply by xcfuu** [reply]
![image](https://user-images.githubusercontent.com/29519964/222305932-c58abb3d-3f5e-4db5-8c1f-7a3ab06b7a6d.png)
没啥报错日志

**Reply by RongtongJin** [reply]
> ![image](https://user-images.githubusercontent.com/29519964/222305932-c58abb3d-3f5e-4db5-8c1f-7a3ab06b7a6d.png) 没啥报错日志

WARN的就是报错，看这个由于某些原因，Slave和Matser频繁断连。。看看Master的store日志吧

**Reply by xcfuu** [reply]
![image](https://user-images.githubusercontent.com/29519964/222317286-8d7967c4-0aaf-48a0-8e88-c448a957074d.png)
master日志

**Reply by RongtongJin** [reply]
@xcfuu 看日志master-slave并不是完全没有同步，但存在频繁断链的情况。如果主备间网络没有问题的话，我看你的版本4.2.0-incubating，在这之后还是修过不少bug的，建议进行升级。

**Comment by mxsm** [comment]
@xcfuu  你看一下Slave下面消费进度的json文件，是否和Master的一样。

**Reply by xcfuu** [reply]
主节点
![image](https://user-images.githubusercontent.com/29519964/222307552-e332f5ab-3b29-4828-ac37-1909e0d1f55c.png)
从节点
![image](https://user-images.githubusercontent.com/29519964/222307751-3a4541b8-1c0f-4d90-bf1c-199fd4317095.png)

**Reply by xcfuu** [reply]
看着进度是一致的

**Comment by caigy** [comment]
@xcfuu 分别进入主、备节点，执行如下命令，对比下结果
`sh bin/mqadmin brokerstatus -b {your broker address} | grep msgGet`

**Reply by xcfuu** [reply]
主 
msgGetTotalTodayMorning         : 19764858
msgGetTotalTodayNow             : 19828834
msgGetTotalYesterdayMorning     : 19621677

从
msgGetTotalTodayMorning         : 0
msgGetTotalTodayNow             : 0
msgGetTotalYesterdayMorning     : 0

**Reply by mxsm** [reply]
> @xcfuu 分别进入主、备节点，执行如下命令，对比下结果 `sh bin/mqadmin brokerstatus -b {your broker address} | grep msgGet`

他这用命令和界面应该是一样的

**Reply by fade-away** [reply]
这个问题有解决吗？我也遇到相同的问题了
通过这个命令从节点拿到的消息数和主节点的不一致
